### ISSUE TITLE
Frontend UI interface of chat task not loading properly for custom bundles

### ISSUE THREAD

**Issue by vaibhavad** [facebookresearch/Mephisto#742]
Hi,

Thank you for releasing Mephisto 1.0.0. I am currently trying to set up a [chat task](https://github.com/facebookresearch/Mephisto/tree/main/examples/parlai_chat_task_demo) but I am facing UI issues when I try to launch the task with custom bundles.

From within `Mephisto/examples/parlai_chat_task_demo`, I first use the following command
```
python parlai_test_script.py
```

On opening the link from the console logs, the interface looks something like this
![Screen Shot 2022-04-01 at 6 34 06 PM](https://user-images.githubusercontent.com/32997732/161350564-2d833698-8901-47dd-99a8-3f4244a205fc.png)

To try a custom build, I first run` npm install; npm run dev` from the`webapp` folder, then I run the following command
```
python parlai_test_script.py conf=custom_prebuilt
```

Now the UI interface gets messed up and looks like this
![Screen Shot 2022-04-01 at 6 35 56 PM](https://user-images.githubusercontent.com/32997732/161350696-861f892f-6a8b-42e5-b9de-dd20a2559c98.png)

The issue is reproducible on both Safari and Chrome web browsers. Neither the frontend nor the backend console shows any error.

Mentioning all relevant versions here
```
python --version
Python 3.7.13
npm -v
8.5.0
nvm -v
0.39.1
node -v
v16.14.2
```

From inside the `webapp` folder
```
npm ls mephisto-task
parlai-mturk-task-compiler@1.0.0 /Users/vaibhav.adlakha/ComplexQA-Mephisto/examples/complex_qa_task/webapp
└─┬ bootstrap-chat@2.0.0
  └── mephisto-task@2.0.0
```

I think this is a version issue. If the custom bundle interface on your end looks fine, can you let me know versions of npm, nvm, and node that you are using?

**Comment by pringshia** [comment]
Thanks for flagging @vaibhavad. I am able to reproduce, looking into it.

**Comment by pringshia** [comment]
Thanks for flagging this @vaibhavad - I have a fix up at #743.

To unblock yourself for now, you can run the following in the webapp directory + rebuild:

```
npm install style-loader@^1.3.0 --save-dev
```
